api/v1alpha1/release_types.go

```diff
@@ -60,6 +60,11 @@ func (rr ReleaseReason) String() string {
 	return string(rr)
 }
 
+const (
```

Comment:
I think this makes more sense in releaseplan. A release itself will never be marked with the `auto-release` label

Reply:
It's something that belongs to the release service. Although I agree, I don't want to duplicate the declaration (actually you cannot as they share the package) and I don't want only one of the two new CRDs to have it defined in their file. Alternatives?

Reply:
What do you mean by duplicate the declaration? Why can't we just move lines 63-66 to the releaseplan_types.go file?

Reply:
Because you need them in the releaseplanadmission as well.

Reply:
Oh I follow now. I can't think of any nice solution. I agree it isn't great to have it in just one of the two new CRDs, but I think having it in the release crd is even worse